microsoft-edge/devtools-guide-chromium/customize/shortcuts.md

```diff
@@ -42,25 +42,25 @@ For example, to pause or continue running a script in Visual Studio Code, you pr
 
 1. To open DevTools, right-click the webpage, and then select **Inspect**.  Or, press **Ctrl+Shift+I** (Windows, Linux) or **Command+Option+I** (macOS).  DevTools opens.
 
-1. In DevTools, on the main toolbar, click the **Settings** (![Settings icon](./shortcuts-images/settings-gear-icon-light-theme.png)) button.  Or, press **Shift+?**.
+1. In DevTools, in the **Activity Bar**, click the **Customize and control DevTools** (![Customize and control DevTools icon](./shortcuts-images/customize-icon.png)) button, and then click the **Settings** (![Settings icon](./shortcuts-images/settings-icon.png)) button.  Or, press **F1**.
 
 1. In the **Settings** panel, select the **Shortcuts** page.
 
 1. Select the action you want to customize.  For example, in the **Debugger** section, select the **Pause script execution** action.
 
-1. Click the **Edit** (![EditKeyboardShortcut](./shortcuts-images/edit-keyboard-shortcut-icon.png)) icon.
+1. Click the **Edit** (![EditKeyboardShortcut](./shortcuts-images/edit-keyboard-shortcut-icon.png)) icon:
```

Comment:
```suggestion
1. Click the **Edit** (![Edit Keyboard Shortcut icon](./shortcuts-images/edit-keyboard-shortcut-icon.png)) icon:
```
adding space chars in this alt-text

Reply:
pushed a commit to fix all alttext in this article and to correct the tooltips and naming of icon buttons.
can Resolve.